phpBB/styles/prosilver/theme/pm.css

```diff
@@ -0,0 +1,87 @@
+/* -------------------------------------------------------------- /*
```

Comment:
Remove the header. Everyone understands that `pm.css` is about PMs.

Reply:
no this is a standard practice, but it should just be "Styles for the PM Module"